bin/testRelease.sh

```diff
@@ -17,4 +17,4 @@ coursier fetch \
   com.lihaoyi:utest_sjs1.0.0-M7_2.11:$VERSION \
   com.lihaoyi:utest_sjs1.0.0-M7_2.12:$VERSION \
   com.lihaoyi:utest_sjs1.0.0-M7_2.13.0-RC1:$VERSION \
-  -r sonatype:releases
+  -r sonatype:public
```

Comment:
I don't know what this does, and it doesn't seem to be done in the history of this repo for new releases, so I can't vet this change.

Reply:
The `Resolver.sonatypeRepo("releases")` resolver has been replaced with `Resolver.sonatypeRepo("public")`. The script uses this resolver so we don't have to wait for Maven Central sync to validate the release succeeded